fastlane/version_utils.rb

```diff
@@ -0,0 +1,29 @@
+module VersionUtils
```

Comment:
@zwolf feel free to review the whole PR. But this is the only Ruby file (besides the fastfiles which I think are Ruby derivative